### PR TITLE
Updated basic_train.ipynb: plot_lr()

### PR DESCRIPTION
**Pull request by rdutta1999**
- [ ] Add details about your PR.
Added some more information about the ```learner``` method ```plot_lr``` in basic_train.ipynb

### CODE REVIEW

File: docs_src/basic_train.ipynb [hunk 1]
```diff
@@ -2863,6 +2863,13 @@
     "show_doc(Recorder.plot_lr)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The learning rate and momentum shown are that of the last layer (```opt.lr``` and ```opt.mom```)."
```
Comment (sgugger) [inline docs_src/basic_train.ipynb:2870]:
```suggestion
    "The learning rate and momentum shown are that of the last layer (`opt.lr` and `opt.mom`)."
```

Comment (sgugger) [reply docs_src/basic_train.ipynb:2870]: Note that single back quotes are the use in the documentation notebooks.

**Comment by sgugger** [comment]
Thanks!

File: docs_src/basic_train.ipynb (hunk 1 above)
Comment (rdutta1999) [reply docs_src/basic_train.ipynb]: Got it. Will keep this in mind for future PRs. Thanks.

**Pull request merged**